src/sequelize/migrations/20190805003447-create-user.js

```diff
@@ -0,0 +1,39 @@
+export const up = (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: Sequelize.INTEGER,
+  },
+  firstName: {
+    allowNull: false,
+    type: Sequelize.STRING,
+  },
+  lastName: {
+    allowNull: false,
+    type: Sequelize.STRING,
+  },
+  userName: {
+    allowNull: false,
+    type: Sequelize.STRING,
+    unique: true,
+  },
+  email: {
+    allowNull: false,
+    type: Sequelize.STRING,
+    unique: true,
+  },
+  password: {
+    allowNull: false,
+    type: Sequelize.STRING,
+  },
+  role: {
+    allowNull: false,
+    type: Sequelize.STRING,
+    defaultValue: 'user',
+  },
+}, {
+  timestamps: false,
```

Comment:
Why didn't you use timestamp? Because I think it would be better to know the time a user started working with our system.

Reply:
According to the PO, the field that we have now are the required one. but since you seam to have a point. We're gonna bring this to the PO's attention